prime-dotnet-webapi/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "Jwt": {
     "Audience": "prime-web-api",
-    "WellKnown": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/v4mbqqas/.well-known/openid-configuration"
+    "WellKnown": "http://localhost:8080/auth/realms/prime/.well-known/openid-configuration"
```

Comment:
Is this correct? How does this work in non localhost environments?

Reply:
I changed this back to the dev instance - it is still not hooked up yet anyway.